src/examples/zkapps/hello_world/run_berkeley.ts

```diff
@@ -0,0 +1,129 @@
+// live Berkeley integration test
+import {
+  Field,
+  PrivateKey,
+  Mina,
+  AccountUpdate,
+  isReady,
+  shutdown,
+  fetchAccount,
+  PublicKey,
+} from 'snarkyjs';
+import { adminPrivateKey, HelloWorld } from './hello_world.js';
+await isReady;
+
+let Berkeley = Mina.Network('https://proxy.berkeley.minaexplorer.com/graphql');
+Mina.setActiveInstance(Berkeley);
+
+let feePayerKey = PrivateKey.random();
+
+console.log(
+  `Funding fee payer ${feePayerKey
+    .toPublicKey()
+    .toBase58()} and waiting for inclusion in a block..`
+);
+
+await faucet(feePayerKey.toPublicKey());
+
+let { nonce, balance } = Berkeley.getAccount(feePayerKey.toPublicKey());
+console.log(
+  `Using fee payer ${feePayerKey
+    .toPublicKey()
+    .toBase58()} with nonce ${nonce}, balance ${balance}`
+);
+
+let zkappKey = PrivateKey.random();
+let zkappAddress = zkappKey.toPublicKey();
+
+let transactionFee = 100_000_000;
+
+console.log('Compiling smart contract..');
+let { verificationKey } = await HelloWorld.compile();
+
+let zkapp = new HelloWorld(zkappAddress);
+let x = await zkapp.x.fetch();
+let isDeployed = x?.equals(0).not().toBoolean() ?? false;
+
+if (!isDeployed) {
```

Comment:
nit: we don't need this check here. there's no way the random public key can already have a deployed zkapp on it